cmd/gokey/main.go

```diff
@@ -12,7 +12,7 @@ import (
 	"strings"
 
 	"github.com/cloudflare/gokey"
-	"golang.org/x/crypto/ssh/terminal"
+	terminal "golang.org/x/term"
```

Comment:
replacing `term` with `terminal` feels backwards

can we just have `term` in the code instead?

Reply:
I thought about it too! Will do `term`